TestAssets/TestProjects/AppsWithNoSln/App/App.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
```

Comment:
We have plenty of test assets like this that you can re-use, instead of adding a new one.